src/NServiceBus.Persistence.DynamoDB.Tests/SagaSchemaVersionTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace NServiceBus.Persistence.DynamoDB.Tests;
```

Comment:
The non block namespace keeps on giving. Can we use the blocked one for now and then move everything over once changes have settled?

Reply:
good catch, will adjust it to be same as the other tests.